### PR TITLE
Infodisplay example

### PR DESCRIPTION
**Pull request by bergie**
* [x] Create participant conforming to [c-base infodisplay interface](https://github.com/c-base/mqttwebview)
* [x] Show URLs in iframe
* [x] Internal rotation when there are no new screens to show
* [x] Send output only after iframe loaded
* [x] Get internal rotation via IIP
* [x] Test the infodisplay via fbp-spec

The fbp-spec tests are a pretty good end-to-end test for whole msgflo-browser, since they require both discovery and regular in/outport communications to work.

### CODE REVIEW

**Comment by bergie** [comment]
Note: Travis tests will require msgflo/msgflo#149

File: index.js [hunk 1]
```diff
@@ -1,4 +1,25 @@
+var querystring = require('querystring');
+
 module.exports = {
   mqtt: require('./src/mqtt'),
   participant: require('./src/participant'),
+  options: function (defaults) {
```
Comment (jonnor) [inline index.js]: Should this allow to not pass any `defaults`? (default to empty object then)

File: package.json [hunk 2]
```diff
@@ -7,19 +7,31 @@
...
   },
   "scripts": {
-    "test": "npm run build",
+    "test": "fbp-spec --address ws://localhost:3569 --command './node_modules/.bin/msgflo --broker=mqtt://localhost' --start-timeout 60 spec/",
     "build": "webpack --config webpack.config.js"
   },
   "author": "",
   "license": "MIT",
   "devDependencies": {
+    "amqplib": "^0.5.1",
```
Comment (jonnor) [inline package.json]: AMQP is not used is it?

Comment (bergie) [inline package.json]: It isn't, but it is a build dep for msgflo

File: index.js (hunk 1 above)
Comment (bergie) [inline index.js]: Fixed in 731f344

**Comment by bergie** [comment]
@jonnor do we want to merge right away, or wait for MsgFlo release to get a green build?

**Comment by jonnor** [comment]
@bergie Wait for green

**Pull request merged**